From 31e45c79aa856943ebee00a772096e655906c926 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Fri, 9 Jan 2026 14:45:39 +0100
Subject: [PATCH 3/4] Handle more settings in optional cmake arg list
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Handle Python_INCLUDE_DIR, Python_LIBRARY, PYBIND11_PYTHON_VERSION
and _PYBIND11_CROSSCOMPILING.

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [oe specific]
---
 hailort/libhailort/bindings/python/platform/setup.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hailort/libhailort/bindings/python/platform/setup.py b/hailort/libhailort/bindings/python/platform/setup.py
index a33a35e..0371b04 100644
--- a/hailort/libhailort/bindings/python/platform/setup.py
+++ b/hailort/libhailort/bindings/python/platform/setup.py
@@ -61,8 +61,12 @@ class build_ext(orig_build_ext):
         "LIBHAILORT_PATH",
         "PYTHON_EXECUTABLE",
         "PYTHON_INCLUDE_DIRS",
+        "Python_INCLUDE_DIR",
         "PYTHON_LIBRARY",
+        "Python_LIBRARY",
         "PYBIND11_FINDPYTHON",
+        "PYBIND11_PYTHON_VERSION",
+        "_PYBIND11_CROSSCOMPILING",
     ]
 
     def run(self):
-- 
2.55.0

